test/GithubApi.Users.test.js

```diff
@@ -0,0 +1,69 @@
+const agent = require('superagent-promise')(require('superagent'), Promise);
+const responseTime = require('superagent-response-time');
+const { expect } = require('chai');
+
+const urlBase = 'https://api.github.com';
+
+describe.only('For getting all users', () => {
+  let queryTime;
+  let usersPagination;
+
+  before(() => {
+    agent
+      .get(`${urlBase}/users`)
+      .auth('token', process.env.ACCESS_TOKEN)
+      .then((response) => {
+        usersPagination = response.body.length;
+      });
+
+    const usersQuery = agent
+      .get(`${urlBase}/users`)
+      .auth('token', process.env.ACCESS_TOKEN)
+      .use(responseTime((request, time) => {
+        queryTime = time;
+      })).then(() => 0);
```

Comment:
please add comment that the .then is necessary for this library because of issues